model_analyzer/config/input/config_value.py

```diff
@@ -237,4 +237,4 @@ def set_name(self, name):
             New name to be set for this field.
         """
 
-        self._name = name
+        self._name = name
```

Comment:
lets just revert the file. shouldn't be any changes.

Reply:
Fixed